Events/src/test/java/com/github/swcraftlyon/meetup/infra/dao/MangoEventDaoTest.java

```diff
@@ -0,0 +1,35 @@
+package com.github.swcraftlyon.meetup.infra.dao;
+
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Filters;
+import org.bson.Document;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class MangoEventDaoTest {
```

Comment:
The point is : the tests don't run green right now because on a given CI server, the prerequisite doesn't exist.
So the code is wrong.
There are several solutions to solve this, but given Mongo is not java, there's no easy one.
I usually use a docker rule (junit @Rule) for that, but it just shift the prerequisite to ... docker.
It's something to discuss I guess.